src/app/ui-components/selectFeeRate/feeItem.tsx

```diff
@@ -160,8 +162,16 @@ function FeeItem({
   const mainColor = totalFee ? 'white_0' : 'white_400';
   const secondaryColor = totalFee ? 'white_200' : 'white_400';
 
+  const feesExceedBalance = Boolean(
+    totalFee && absoluteBalance && absoluteBalance < Number(totalFee),
+  );
```

Comment:
I don't think this is correct. This would assume that you don't have enough funds if the fee is above your balance, disregarding the amount that you're sending.

e.g. If my balance is 1000 and I'm sending 950 with a fee of 100, then this will still show false because 100<1000 when I don't actually have enough balance for this fee (need 1050 but only have 1000)